### ISSUE TITLE
training on my own data

### ISSUE THREAD

**Issue by Flyzzz** [baldassarreFe/deep-koalarization#13]
for my own dataset, I don't have any label , just color images, any instructions for this ? thank you

**Comment by baldassarreFe** [comment]
Class labels are not needed for colorization, in a way the pixel colors represent the training target.

You should be able to set up your own training by following the [report](https://github.com/baldassarreFe/deep-koalarization/blob/master/paper.pdf) and all papers we reference in it. The main steps are: 
- collect colored images (if your dataset is small, consider downloading more images)
- process the images to extract a black and white image that will be used as input and color information that will be used as target
- set up your architecture, anything that looks like a U-net could work, features extracted using a pre-trained inception model can be added to improve the colorization
- train your network by feeding in the grayscale version of an image and comparing the output with the colored version of the image

Please note, this is project is based on an outdated version of tensorflow, so it's fine for taking inspiration for the general pipeline, but when it comes to the code itself I suggest reimplementing everything from scratch.
